src/api/spec/lib/rubocop/cop/view_component/avoid_global_state_spec.rb

```diff
@@ -15,14 +15,18 @@ def initialize
                    ^^^^^^^^^^^^ View components should not rely on global state by using params[:abc]. Instead, pass the required data to the initialize method.
             @abc_def = params[:abc][:def]
                        ^^^^^^^^^^^^ View components should not rely on global state by using params[:abc]. Instead, pass the required data to the initialize method.
+            @def = User.session
+                   ^^^^^^^^^^^^ View components should not rely on global state by calling User.session. Instead, pass the required data to the initialize method.
+            @aaa = User.find_by(login: 'someone')
+                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ View components should not rely on global state by calling User.find_by(login: 'someone'). Instead, pass the required data to the initialize method.
 
           end
         end
       RUBY
     end
   end
 
-  context 'when a view component does not use params' do
+  context 'when a view component does not use params or call a User class method' do
```

Comment:
```suggestion
  context 'when a view component does not use params and does not call a User class method' do
```

Reply:
The meaning is the same, but if you prefer this version, no worries!